stetho/src/main/java/com/facebook/stetho/inspector/elements/Descriptor.java

```diff
@@ -55,6 +58,44 @@ public final void postAndWait(Runnable r) {
     getHost().postAndWait(r);
   }
 
+  public void setAttributesAsText(Object element, String text) {
```

Comment:
But how am I supposed to call `mSuper.setAttributesAsText(element, text);` on `ChainedDescriptor` if `Descriptor` doesn't implement that method?